pkg/chains/formats/slsa/v2alpha2/internal/pipelinerun/pipelinerun.go

```diff
@@ -94,6 +94,15 @@ func internalParameters(pro *objects.PipelineRunObject) map[string]any {
 // externalParameters adds the pipeline run spec
 func externalParameters(pro *objects.PipelineRunObject) map[string]any {
 	externalParams := make(map[string]any)
+	// add top level pipeline config
+	if p := pro.Status.Provenance; p != nil && p.RefSource != nil && pro.Spec.PipelineRef != nil {
+		buildConfigSource := map[string]any{
+			"ref":        p.RefSource.Digest,
```

Comment:
p.RefSource.Digest is a map field, but I suppose the intention for the `ref` field should be `string` according to the discussion in https://github.com/tektoncd/chains/issues/846#issuecomment-1611835123 and also slsa recommendation for GHA https://github.com/slsa-framework/github-actions-buildtypes/tree/main/workflow/v1#external-parameters.

Maybe we need to grab the specific digest from the map (also there should be always only one pair there).

Reply:
should I always set it to a string as `sha1:abc123` or `sha256:abc123`?

Reply:
SGTM!

Reply:
done.